rust/processor/src/db/common/models/object_models/raw_v2_objects.rs

```diff
@@ -126,7 +128,7 @@ impl RawObject {
         txn_version: i64,
         write_set_change_index: i64,
         object_mapping: &AHashMap<CurrentObjectPK, RawCurrentObject>,
-        conn: &mut DbPoolConnection<'_>,
+        conn: &mut Option<DbPoolConnection<'_>>,
```

Comment:
Spoke offline; we thought it doesn't make sense to have a required `query_retries` and `q_retry_delay` when the client isn't present. At the same time, having all as option could lead to some user error. Agreed on introducing a new struct `DbConnection` as an Optional parameter with all three fields as required.